openhtf/core/measurements.py

```diff
@@ -139,6 +140,19 @@ def _initialize_value(self):
     else:
       self.measured_value = MeasuredValue(self.name)
 
+  def __deepcopy__(self, memo):
```

Comment:
How's this?  I only call _initalize_value() if measured_value is not passed in.  More intuitive to only initialize if value is not passed vs initializing while sidestepping ValueError then overriding.